oap-server/pom.xml

```diff
@@ -89,6 +89,7 @@
         <freemarker.version>2.3.28</freemarker.version>
         <javaassist.version>3.25.0-GA</javaassist.version>
         <vavr.version>0.10.3</vavr.version>
+        <groovy.version>3.0.3</groovy.version>
```

Comment:
Check `check-LICENSE.sh` and `known-oap-backend-dependencies.txt`/`known-oap-backend-dependencies-es7.txt`. You need to update LICENSE/NOTICE and license check process. You are bringing a new like, maybe including many jars into OAP.